theblues/charmstore.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 import requests
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, RequestException, Timeout
```

Comment:
Previously I didn't use that format unless the imports ran over the 80 char width limit. Is that standard for all instances of multiple imports from the same module?